Can't get it to compile in XCode 4.6

When compiling the release version of WTF iOS for iPhone 6.0 Simulator, I get errors in FastMalloc.cpp.  Any idea how to fix the errors?

.../XCode Projects/JavaScriptCore-iOS/WTF/wtf/FastMalloc.cpp:2477:8: error: private field 'pad_' is not used [-Werror,-Wunused-private-field]
  char pad_[(64 - (sizeof(TCMalloc_Central_FreeList) % 64)) % 64];

.../XCode Projects/JavaScriptCore-iOS/WTF/wtf/FastMalloc.cpp:4544:31: error: private field 'm_reader' is not used [-Werror,-Wunused-private-field]
    const RemoteMemoryReader& m_reader;

--- a/WTF/WTF.xcodeproj/project.pbxproj
+++ b/WTF/WTF.xcodeproj/project.pbxproj
@@ -1754,6 +1754,7 @@
 			baseConfigurationReference = 5D247B7314689C4700E78B76 /* WTF.xcconfig */;
 			buildSettings = {
 				DEBUG_DEFINES = "$(DEBUG_DEFINES_debug)";
+				GCC_TREAT_WARNINGS_AS_ERRORS = NO;
 			};
 			name = Debug;
 		};
@@ -1761,6 +1762,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 5D247B7314689C4700E78B76 /* WTF.xcconfig */;
 			buildSettings = {
+				GCC_TREAT_WARNINGS_AS_ERRORS = NO;
 			};
 			name = Release;
 		};
@@ -1775,6 +1777,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 5D247B7314689C4700E78B76 /* WTF.xcconfig */;
 			buildSettings = {
+				GCC_TREAT_WARNINGS_AS_ERRORS = NO;
 			};
 			name = Production;
 		};
@@ -1791,6 +1794,7 @@
 					"$(GCC_PREPROCESSOR_DEFINITIONS)",
 					"UCONFIG_NO_COLLATION=1",
 				);
+				GCC_TREAT_WARNINGS_AS_ERRORS = NO;
 				IPHONEOS_DEPLOYMENT_TARGET = 5.0;
 				PRODUCT_NAME = WTF;
 				SDKROOT = iphoneos;
@@ -1810,6 +1814,7 @@
 					"$(GCC_PREPROCESSOR_DEFINITIONS)",
 					"UCONFIG_NO_COLLATION=1",
 				);
+				GCC_TREAT_WARNINGS_AS_ERRORS = NO;
 				IPHONEOS_DEPLOYMENT_TARGET = 5.0;
 				PRODUCT_NAME = WTF;
 				SDKROOT = iphoneos;
@@ -1829,6 +1834,7 @@
 					"$(GCC_PREPROCESSOR_DEFINITIONS)",
 					"UCONFIG_NO_COLLATION=1",
 				);
+				GCC_TREAT_WARNINGS_AS_ERRORS = NO;
 				IPHONEOS_DEPLOYMENT_TARGET = 5.0;
 				PRODUCT_NAME = WTF;
 				SDKROOT = iphoneos;
@@ -1898,6 +1904,7 @@
 				B6E69982166BD499005EF4B1 /* Production */,
 			);
 			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Production;
 		};
 /* End XCConfigurationList section */
 	};

